### PR TITLE
Note that this project isn't affiliated with upstream

### PR DESCRIPTION
**Pull request by mara004**
CC @bblanchon @M-Waszkiewicz-Anaconda @boldorider4 @skupr-anaconda

### CODE REVIEW

**Comment by mara004** [comment]
@ Anaconda Team: We'd prefer if you would please remove this package from the main and anaconda channels, to avoid confusion. (Similar to how pypdfium2-feedstock exists but isn't published to main.)

**Comment by boldorider4** [comment]
Hi @mara004,
we can discuss internally moving the package to a custom channel instead of main, but we cannot remove the feedstock. Although this feedstock is not affiliated with the upstream project, it is pulling the binaries from `bblanchon::pdfium-binaries`. We need this because the way we build the dependent [feedstock](https://github.com/AnacondaRecipes/pypdfium2-feedstock/blob/1077be212ff57c56f5f523f9c07535961dff606d/recipe/meta.yaml#L29C1-L29C1).

File: recipe/meta.yaml [hunk 1]
```diff
@@ -73,9 +73,10 @@ test:
 
 about:
   home: https://github.com/bblanchon/pdfium-binaries
-  summary: pre-compiled binaries of the PDFium library
+  summary: pdfium-binaries (repack by Anaconda Inc.)
```
Comment (boldorider4) [inline recipe/meta.yaml:76]:
```suggestion
  summary: repack of [pdfium-binaries](bblanchon::pdfium-binaries)
```

File: recipe/meta.yaml [hunk 2]
```diff
@@ -73,9 +73,10 @@ test:
...
   description: |
-    This project hosts pre-compiled binaries of the PDFium library, an open-source library for PDF manipulation and rendering.
+    Pre-compiled binaries of the PDFium library, an open-source library for PDF manipulation and rendering.
+    Note: This is a third-party repack. It is maintained by Anaconda Inc., not pdfium-binaries publisher Benoît Blanchon. Upstream provides its own conda builds at bblanchon::pdfium-binaries.
```
Comment (boldorider4) [inline recipe/meta.yaml:79]:
```suggestion
    This repack, maintained by Anaconda Inc., takes binaries from upstream project at bblanchon::pdfium-binaries. Upstream project also provides their own conda builds.
```

**Comment by mara004** [comment]
> Hi @mara004, we can discuss internally moving the package to a custom channel instead of main, but we cannot remove the feedstock. Although this feedstock is not affiliated with the upstream project, it is pulling the binaries from `bblanchon::pdfium-binaries`. We need this because the way we build the dependent [feedstock](https://github.com/AnacondaRecipes/pypdfium2-feedstock/blob/1077be212ff57c56f5f523f9c07535961dff606d/recipe/meta.yaml#L29C1-L29C1).

Hi @boldorider4,

the feedstock repo is OK and there's no need to remove it. We would just like the *resulting packages* to be removed from default channels (main/anaconda) to avoid confusion. If callers do `conda install pdfium-binaries` they'd currently get the unofficial repack, which is a bit problematic. That's why I believe it would be best to remove these builds from main and have people specify the channel they want explicitly instead. So PR #8 looks good.

Thanks!

PS: Since pypdfium2 also has official conda packages now, I plan to make a similar PR to the pypdfium2-feedstock updating its Readme.

**Comment by mara004** [comment]
Also, feel free to apply the suggestions (assuming maintainers have this access).
It's just the gist to have a notice, I don't care too much about the details.

File: recipe/meta.yaml (hunk 1 above)
Comment (mara004) [reply recipe/meta.yaml:76]: Will this kind of link actually work in the summary?

File: recipe/meta.yaml (hunk 2 above)
Comment (mara004) [reply recipe/meta.yaml]: I pushed something new based on your suggestion.

**Comment by mara004** [comment]
@remkade See my previous comment https://github.com/AnacondaRecipes/pdfium-binaries-feedstock/pull/7#issuecomment-1788114112, but I can elaborate: The goal is mainly to avoid confusion and point out to users which builds are official and which are managed by a third party (in this case, anaconda). Part of the problem is that this feedstock currently publishes to main, but our packages need the `bblanchon::pdfium-binaries` instead, which adds to the confusion.
If there's no notice, this implicitly gives the package an official touch, people might not realize it's a third party's work.

Also it's no secret we as upstream would prefer people to use our official builds now they exist.
We can't really provide support or debugging help for third-party builds, they're just out of our hands.
Also packaging pypdfium2 correctly is a bit complicated (ABI/API safety while maintaining dependency flexibility, supporting all platforms, regular updates, etc.), and we don't want to rely on third parties here.

To sum up,
* we provide our own packages, which many other projects don't, so this notice would help avoid confusion
* since I'm lead dev of pypdfium2 (for which you created these recipes) I'd hope my preference here would be respected.

Thanks.

**Comment by remkade** [comment]
Makes sense, thanks for the context. We're planning on also clarifying in our installers and other places that we are not affiliated with the upstream projects in any way, so hopefully that also helps in addressing these concerns.

@boldorider4 Lets go ahead and merge these if they're ready to go.

**Comment by remkade** [comment]
Lets talk further internally on the prospect of moving these to another channel. That part has a lot more complicated implications.

**Comment by mara004** [comment]
Thanks a lot for the cooperation!

I understand what I've asked for may be a bit odd, seeing as Linux distros have been packaging for ages without this kind of clarification (and I suppose it's perfectly legal as long as the package name is not a registered trademark), but my personal opinion is that it might help upstreams and users sometimes.

**Comment by mara004** [comment]
@boldorider4 Could you please also merge the equivalent PR for the pypdfium2 feedstock itself?https://github.com/AnacondaRecipes/pypdfium2-feedstock/pull/3
